contrast_security/user_api/application_api.py

```diff
@@ -151,7 +151,7 @@ def filter_application_traces(self, org_uuid, app_id, application_trace_filter=N
         if application_trace_filter is None:
             application_trace_filter = ApplicationTraceFilter()
         path = '{org_uuid}/traces/{app_id}/filter'.format(org_uuid=org_uuid, app_id=app_id)
-        return self._post(path, params=application_trace_filter.get_params_as_json())
+        return self._get(path, params=application_trace_filter.get_params_as_json())
```

Comment:
same for this endpoint
https://github.com/Contrast-Security-Inc/teamserver/blob/79bca0b6033cbcd3a5e639baf8b7475afb9fca57/teamserver-app/src/main/java/contrast/teamserver/rest/ng/trace/NgApplicationTraceListingRestController.java#L227

Reply:
see reply above